shared/src/packages/users/libs/types/user-sign-up-response-dto.type.ts

```diff
@@ -1,8 +1,8 @@
+import { type UserRegisterResponse } from './user-register-response.type.js';
+
 type UserSignUpResponseDto = {
-  id: number;
-  email: string;
-  firstName: string;
-  lastName: string;
+  token: string;
+  user: UserRegisterResponse;
```

Comment:
```suggestion
  user: UserAuthResponse;
```
for sign-in and sign-up should be the same type for the user